src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -1136,14 +1136,8 @@ def getWorkloadAssignArgs():
                                            "validate": lambda x: all([cmsname(y) for y in x])},
                      "AutoApproveSubscriptionSites": {"default": [], "type": makeList, "assign_optional": True,
                                                       "validate": lambda x: all([cmsname(y) for y in x])},
-                     "CustodialSubType": {"default": "Replica", "type": str, "assign_optional": True,
-                                          "validate": lambda x: x in ["Move", "Replica"]},
-                     "NonCustodialSubType": {"default": "Replica", "type": str, "assign_optional": True,
-                                             "validate": lambda x: x in ["Move", "Replica"]},
-                     "CustodialGroup": {"default": "DataOps", "type": str, "assign_optional": True},
-                     "NonCustodialGroup": {"default": "DataOps", "type": str, "assign_optional": True},
                      "SubscriptionPriority": {"default": "Low", "assign_optional": True,
-                                              "validate": lambda x: x.lower() in PhEDEx_VALID_SUBSCRIPTION_PRIORITIES},
+                                              "validate": lambda x: x.lower() in RUCIO_RULES_PRIORITY.keys()},
```

Comment:
this line could be simply:
```
"validate": lambda x: x.lower() in RUCIO_RULES_PRIORITY},
```

Reply:
Done with https://github.com/dmwm/WMCore/pull/11181/commits/edb3ee5780ddf03634821fa190f0a35db9d51895